argocd-notifications/README.md

```diff
@@ -0,0 +1,81 @@
+# Argocd-Notifications
```

Comment:
Let's move this to: https://github.com/operate-first/apps/tree/master/docs/content/argocd-gitops 

And add it to the jupyterbook, instructions here: https://www.operate-first.cloud/apps/CONTRIBUTING.html

Reply:
The purpose of this doc isn't clear to me starting out, why should I read this doc? What sort of information do we expect users to seek? In my mind the doc should answer this question: 

> Where and How do I add notifications to my ArgoCD apps?

Reply:
I see that you want to add additional descriptions for Triggers. So maybe separate the doc into 2 parts: 

> How to add notifications
   * step by step instruction on how to trigger a notification for an argocd app in the `apps` repo, what files need to be updated? where? etc. 
   
> References
* describe further if you feel that it's needed argocd-notification concepts like triggers/templates/etc. and you can refer to these in the above section should the need arise.